MarkdownEditor.elm

```diff
@@ -0,0 +1,39 @@
+module MarkdownEditor where
+-- Elm’s "import" keyword works mostly like "require" in node.js.
```

Comment:
i think its good to leave them in there while we get our bearings just because... i'll be referencing the comments alot to remember what the stuff does. agree though that at some point we can remove